conways-game-of-life/src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
```

Comment:
Just another suggestion as well to remove the code you are not using like the serviceWorker code and file or any other files that create-react-app automatically generates that you don't need

Reply:
actually I have not any Idea about this seviceWorker file